src/Bicep.Core/Configuration/ConfigHelper.cs

```diff
@@ -39,7 +37,7 @@ private IConfigurationRoot BuildConfig(string? localFolder)
             // load the default settings from file embedded as resource
             var assembly = Assembly.GetExecutingAssembly();
             var names = assembly.GetManifestResourceNames();
-            var defaultConfigResourceName = names.FirstOrDefault(n => n.EndsWith(SettingsFileName));
+            var defaultConfigResourceName = names.FirstOrDefault(n => n.EndsWith(LanguageConstants.BicepConfigSettingsFileName));
```

Comment:
I know this is existing code, but I think it would be better to be precise here and hard-code the manifest resource path.

Reply:
Updated here:
https://github.com/Azure/bicep/commit/69925779a6420b1c5b57f82f7aea7125dad6538b